templates/packer/worker.pkr.hcl

```
@@ -55,6 +55,38 @@ build {
     }
   }
 
+  source "amazon-ebs.image_builder" {
+    name = "rhel-8-aarch64"
+
+    # Use a static RHEL 8.6 Cloud Access Image.
+    source_ami = "ami-0c84d76d81209a0e2"
+    ssh_username = "ec2-user"
+    instance_type = "c6g.large"
+
+    # Set a name for the resulting AMI.
+    ami_name = "${var.image_name}"
+
+    # Apply tags to the resulting AMI/EBS snapshot.
+    tags = {
+      AppCode = "IMGB-001"
+      Name = "${var.image_name}"
+      # A different tag from x86_64 to help find the right AMI
+      # Ideally, 'arch' & 'composer_commit' tags should be used instead.
+      composer_commit_aarch64 = "${var.composer_commit}"
```

As soon as https://github.com/app-sre/qontract-reconcile/pull/2733 has been merged and rolled out, you can use the tag name `composer_commit` too.

```suggestion
      composer_commit = "${var.composer_commit}"
```

---

Usage of it can be seen in this [app-interface MR](https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/46049/diffs)